### PR TITLE
feat: support ipv6

### PR DESCRIPTION
**Pull request by czybjtu**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -12,7 +12,7 @@ export GOROOT GO111MODULE CGO_ENABLED
 
 
 build:
-	go build -v -o "$(OUT_DIR)/$(KIND_CLOUD_BINARY_NAME)" $(KIND_CLOUD_BUILD_FLAGS) cmd/main.go
+	go build -o "$(OUT_DIR)/$(KIND_CLOUD_BINARY_NAME)" $(KIND_CLOUD_BUILD_FLAGS) cmd/main.go
```
Comment (BenTheElder) [inline Makefile:15]: Unrelated change?

Comment (czybjtu) [reply Makefile:15]: yes. I think it's unnecessary to print so much compilation infomation. I can checkout it if you think it's not appropriate.

Comment (BenTheElder) [reply Makefile:15]: At the very least this should be in a different commit.

I think it's useful to print this, personally.

Comment (czybjtu) [reply Makefile]: Have been checkouted.

**Comment by aojea** [comment]
Thanks, have you tested it?

**Comment by czybjtu** [comment]
Yes. I have tested the following scenarios:
- IPv4 single stack
- IPv6 single stack
- dualstack

**Comment by aojea** [comment]
/lgtm
/approve

TODO: add tests

**Comment by aojea** [comment]
Thanks very much
